tests/res_block_test.py

```diff
@@ -1,13 +1,9 @@
-
 import tensorflow as tf
 from tensorflow import keras
 import numpy as np
 import unittest
-
 from lib import res_block
 
```

Comment:
Style: Each class and global-space function should be preceded by two blank lines.